packages/dd-trace/test/appsec/iast/context/context-plugin.spec.js

```diff
@@ -68,8 +67,8 @@ describe('IastContextPlugin', () => {
     it('should add a subscription to the channel', () => {
       plugin.startCtxOn(channelName, tag)
 
-      expect(addSub).to.be.calledOnceWith(channelName)
-      expect(getAndRegisterSubscription).to.be.calledOnceWith({ channelName, tag, tagKey: TagKey.SOURCE_TYPE })
+      sinon.assert.calledOnceWithMatch(addSub, channelName)
+      sinon.assert.calledOnceWithExactly(getAndRegisterSubscription, { channelName, tag, tagKey: TagKey.SOURCE_TYPE })
```

Comment:
nit: why is one converted to calledOnceWithMatch and the other to calledOnceWithExactly ? just because otherwise the test fails ?